### PR TITLE
fix: fix error link

### PR DESCRIPTION
**Pull request by caicancai**

### CODE REVIEW

**Comment by caicancai** [comment]
@chunshao90 If you have time, can you help me review this small change? Thank you.

**Comment by caicancai** [comment]
cc @chunshao90 Good to merge

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ PR title.
 
 For PRs that you consider ready for review, verify the following locally before you submit it:
 
-* you have a coherent set of logical commits, with messages conforming to the [Conventional Commits](https://github.com/apache/incubator-horaedb/docs/conventional-commit.md) specification;
+* you have a coherent set of logical commits, with messages conforming to the [Conventional Commits](https://github.com/apache/incubator-horaedb-docs/blob/029e993f359907b7502b969a1cffeb8e43cb17fb/docs/src/en/dev/conventional_commit.md) specification;
```
Comment (ShiKaiWi) [inline CONTRIBUTING.md:40]: I guess this is the link we want: `https://github.com/apache/incubator-horaedb-docs/blob/main/docs/src/en/dev/conventional_commit.md`.

**Comment by ShiKaiWi** [comment]
@caicancai Sorry for late reply. The notification bot doesn't work after the project is donated to ASF, and attention is not paid to this PR in time for this. The bot was fixed yesterday, so we find you have closed the PR. Truly sorry for such thing.

Would you like to submit another PR with the right link I mention in the review comments?

BTW, would you like to join into our slack channel? And in the channel, you can talk to all the committees and notify them of the PRs to review.

**Pull request closed without merging**